src/config/migrations.rs

```diff
@@ -87,11 +87,24 @@ impl From<NetworkConfigV1> for NetworkConfigV2 {
     }
 }
 
+pub fn update_config_v2_to_v2_1(config_v2: ConfigV2) -> ConfigVersion {
+    let mut config_v2_1 = config_v2;
+    for (name, network_config) in config_v2_1.network_connection.iter_mut() {
+        if name == "testnet" && network_config.fastnear_url.is_none() {
+            network_config.fastnear_url = Some("https://test.api.fastnear.com/".parse().unwrap());
+        }
+    }
+    ConfigVersion::V2_1(config_v2_1)
+}
+
 #[derive(serde::Serialize, serde::Deserialize)]
 #[serde(tag = "version")]
 pub enum ConfigVersion {
     #[serde(rename = "1")]
     V1(ConfigV1),
     #[serde(rename = "2")]
     V2(ConfigV2),
+    // Adds fastnear_url to the testnet config if it's not present
+    #[serde(rename = "2.1")]
+    V2_1(ConfigV2),
```

Comment:
Not sure if adding a minor/patch version for configs is a good idea. I'd keep only major version incremental like `v3`, `v4` and so on, but this is not very critical, so if you have a strong opinion that it should be `v2.1`, then let's keep it that way

Reply:
We don't have any changes in the fields that's why I decided to go with a minor version. But I don't have strong opinion as well

Reply:
Also, I think update should be done with `impl From<ConfigV2> for ConfigV2_1`, not with a regular function